sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathAccessControlEntry.java

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.common.implementation.StorageImplUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * POSIX access control rights on files and directories. The value is a comma-separated list of access control entries.
```

Comment:
Could this be broken into a couple paragraphs and if needed have lists to show examples? For example

```
PathAccessControlEntry represents POSIX access control rights on files and directories.

<p>
The value is a comma-separated list of access control entries, each access control entry (ACE) consists of four elements:
<ul>
  <li>Scope</li>
  <li>Type</li>
  <li>User or Group Identifier</li>
  <li>Permissions</li>
</ul>
```

Right now this is a wall of text which has key information that could be lost in its density.